reflex/components/forms/editable.py

```diff
@@ -36,6 +38,31 @@ class Editable(ChakraComponent):
     # The initial value of the Editable in both edit and preview mode.
     default_value: Var[str]
 
+    # Time in milliseconds to wait between end of input and triggering on_change
+    debounce_timeout: Var[int]
+
+    @classmethod
+    def create(cls, *children, **props) -> Component:
+        """Create an Editable component.
+
+        Args:
+            children: The children of the component.
+            props: The properties of the component.
+
+        Returns:
+            The component.
+        """
+        if (
+            isinstance(props.get("value"), Var) and props.get("on_change")
+        ) or "debounce_timeout" in props:
+            # Create a debounced input if the user requests full control to avoid typing jank
+            return DebounceInput.create(
+                super().create(*children, **props),
+                # Currently default to 50ms, which appears to be a good balance
+                debounce_timeout=props.pop("debounce_timeout", 50),
```

Comment:
have to `pop` the prop before passing off to `super().create` on line 60

Reply:
oh oops, this happens after the super.create, I'll fix it.

Reply:
fixed in https://github.com/reflex-dev/reflex/pull/1650/commits/80ce5fb85e003d5872043d5fb943bd77b77c6833